tests/EventSubscriber/CronMonitorSubscriberTest.php

```diff
@@ -86,14 +87,22 @@ public function testOnConsoleCommandStartDisabled()
         $eventDispatcher->addSubscriber(new CronMonitorSubscriber($cronMonitorFactory));
 
         // Act
-        $inputDefinition = new InputDefinition();
-        $input = new ArrayInput([], $inputDefinition);
+        $input = $this->createMock(InputInterface::class);
+
         $event = new ConsoleCommandEvent(null, $input, new NullOutput());
 
-        $eventDispatcher->dispatch($event, ConsoleEvents::COMMAND);
+        $input->expects(self::once())
+            ->method('hasOption')
+            ->with('cron-monitor-slug')
+            ->willReturn(false);
 
-        // Assert
-        // Not easily possible with phpunit :( Main assert is in Arrange section
+        $input->expects(self::never())
+            ->method('getOption');
+
+        $cronMonitorFactory->expects(self::never())
+            ->method('create');
```

Comment:
By mocking input you are testing implementation not which is not good approach could just have left empty input.

The $cronMonitorFactory assert is nice indeed butyou could use it in assert section in assert format, I mean:
```
$cronMonitorFactory->shouldNotHaveReceived('create');
```

Reply:
Removed `$input` mocking

As for `shouldNotHaveReceived` it's only available in `mockery/mockery` package and it's not used in this bundle.